scripts/tasks/build.ts

```diff
@@ -9,6 +9,7 @@ export const build: Task = {
     return pathExists(builtSandboxDir);
   },
   async run({ sandboxDir }, { dryRun, debug }) {
+    throw new Error('Problems!');
```

Comment:
No problems please!

Reply:
😂

Reply:
This is how it looks like now

<img width="482" alt="image" src="https://user-images.githubusercontent.com/1671563/200802859-97a22ded-da31-44a4-8d98-1c4ed5fec821.png">